CHANGELOG.md

```
@@ -288,6 +292,11 @@ Format based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 ### Fixed
 
 - **gsd**: promote `CONTEXT-DRAFT.md` to final `CONTEXT.md` in the `gsd_summary_save` tool path (#4442)
+=======
+### Added
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)